ext/hal/inventek/ism43362-m3g-l44/Inc/es_wifi_conf.h

```diff
@@ -6,31 +6,31 @@
   ******************************************************************************
```

Comment:
I'd rather see all this adaptation below in drivers/wifi once you'll be implementing the actual wifi driver.

The idea being: if you need to upgrade the ext/hal, it will be painful to backport your modifications. So, imo, the least amount of modifications should be done in ext/hal, rest of the adaptation in the relevant native part (drivers/, ...).

See what I did here in PR #4711 (original PR was mixing too much modifications in drivers and ext/hal), in the end besides the CMakeLists.txt, I only had to modify one unique file from the whole original ones from ASF.